exp-player/addon/components/exp-physics-intro/template.hbs

```diff
@@ -30,12 +28,6 @@
             select a privacy level for your webcam recording.
         </p>
         <br>
-        <p style="color:blue;">
```

Comment:
> Nope, there were no JS errors whatsoever, and there was HTML being loaded, just not that made for anything showing up. Annnnd now I can't reproduce it, so who knows. I'm sorry!

I actually like when bugs go away on their own! The most common reasons for things to not show up are a build error (eg if you were missing a dependency that got picked up when you restart your ember server), or a slow server (ember pages that depend on JamDB wait to load until the data has arrived, and sometimes the internet gets sluggish for a few minutes). We will monitor for future issues then.

. But it is something to watch for.